### ISSUE TITLE
[tinkerification] Remove scheduler from inside the workers and default to use set_lr() and get_lr() in training loop

### ISSUE THREAD

**Issue by erictang000** [NovaSky-AI/SkyRL#998]
After #978, we should remove the scheduler logic in SkyRL-Train workers and just rely on the tinker convention of setting learning rate for each `optim_step()` call

cc: @tyler-griggs

**Comment by jonahsamost** [comment]
I can take this
